pkg/router/template/router_test.go

```diff
@@ -972,14 +990,104 @@ func TestCalculateServiceWeights(t *testing.T) {
 				suKey1: 1,
 			},
 		},
+		{
+			name:      "a single service with multiple endpoints, but no endpoint ports match the route port",
+			routePort: "9090",
```

Comment:
I changed `routePort` to "port" but the test failed. Shouldn't the symbolic form also pass?

Reply:
In this specific case, it will fail if you change it to "port" because the test matches "8080" or "port" and this test case is testing *not* matching either of those, hence why it's currently "9090". So your changing it to actually match, when we are testing for it to not match.

Reply:
OK, thanks. I went through and fiddled with the inputs getting failures where I understood them and non-failures when substituting <port-number> for "port" -- except this specific case which is why I was seeking some clarification. All good.